src/vuln_analysis/server/webhooks/config.py

```
@@ -0,0 +1,168 @@
+
+    # Outbound auth (generic; no provider named in code).
+    auth_mode: str = AUTH_MODE_NONE
+    auth_token_url: str | None = None
+    auth_client_id: str | None = None
+    auth_client_secret: str | None = None
+    auth_scope: str | None = None
+
+    # SQLite ledger path for restart-durable idempotency. Defaults under the reports root.
+    ledger_path: str | None = None
+
+    def __post_init__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__post_init__`

(ANN204)